app.js

```diff
@@ -3,18 +3,12 @@
 const express = require('express');
 const app = express();
 const morgan = require('morgan');
-const path = require('path');
-module.exports = app; 
+module.exports = app;
 
 app.use(morgan('dev'));
-app.use(express.static(path.join(__dirname, './public')));
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
 app.use('/api', require('./route/index'));
 
-app.get('/', (req, res) => {
-    res.redirect('/puppies')
-})
```

Comment:
This `res.redirect` is a little un-idiomatic in that if the home path is requested, it redirects the client to `/puppies`, which isn't actually a route in your app (your app was serving up at `/api` and my fix changes it to `/api/puppies`). This means that root requests would serve up a 404, so this redirect might as well not exist.